CleverFerret/src/main/java/com/universalmedialibrary/services/network/NetworkExplorerService.kt

```
@@ -43,430 +35,273 @@ class NetworkExplorerService @Inject constructor(
+                    try {
+                        val address = InetAddress.getByName(hostAddress)
+                        if (address.isReachable(100)) {
+                            devices.add(NetworkDevice(
+                                id = hostAddress,
+                                name = address.hostName ?: hostAddress,
+                                ipAddress = hostAddress,
+                                type = DeviceType.UNKNOWN,
+                                services = emptyList()
+                            ))
+                        }
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)